README.md

```diff
@@ -4,6 +4,35 @@
 
 This repository contains the Agent Integrations that Datadog officially develops and supports. To add a new integration, please see the [Integrations Extras](https://github.com/DataDog/integrations-extras) repository and the [accompanying documentation](http://docs.datadoghq.com/guides/integration_sdk/).
 
+
+# Quick development Setup
+
+To get started developing with the integrations-core repo you will need: `gem` and `python`.
+
+on Debian:
+
+```
+apt-get install rubygems python
+```
```

Comment:
let's remove the instructions, it'll be more generic (or we should do the same for different distros/os)